### PR TITLE
BACKLOG-12743 style CE header

### PR DESCRIPTION
**Pull request by Taybou**

### CODE REVIEW

File: src/javascript/DesignSystem/ContentLayout/ContentHeader/ContentHeader.test.jsx [hunk 1]
```diff
@@ -3,51 +3,8 @@ import {shallow} from '@jahia/test-framework';
 import ContentHeader from './ContentHeader';
 
 describe('Content header', () => {
-    it('Should render with everything', async () => {
```
Comment (FBerthelot) [inline src/javascript/DesignSystem/ContentLayout/ContentHeader/ContentHeader.test.jsx]: Nope, don't delete tests

Comment (Taybou) [reply src/javascript/DesignSystem/ContentLayout/ContentHeader/ContentHeader.test.jsx]: I update the component, so I must delete and or update the test :D

Comment (FBerthelot) [reply src/javascript/DesignSystem/ContentLayout/ContentHeader/ContentHeader.test.jsx]: Yes please update the test to the new behaviour.

All test that you delete, you can move them to another place where the unchanged test belong.

Comment (Taybou) [reply src/javascript/DesignSystem/ContentLayout/ContentHeader/ContentHeader.test.jsx]: the is no other places, the code doesn't exist anymore

**Pull request merged**